### ISSUE TITLE
forward `$$restProps` or expose `id` prop or `rootRef`

### ISSUE THREAD

**Issue by huntabyte** [thecodejack/svelte-file-dropzone#162]
I have a button in my app that, when clicked, I want the same behavior as when a user clicks the dropzone.

Currently, we can't pass any additional data-* attributes or `id` to the underlying `<div>` to get a reference to it.

**Comment by roberrrt-s** [comment]
Can I one-up this one.

**Comment by arackaf** [comment]
Yeah def - lemme take a look tonight or tomorrow

**Comment by roberrrt-s** [comment]
My hacky workaround currently is:

```javascript
    function openFileInput() {
        // I hate everything about this but it works.
        const input = document.querySelector('.drop-element input[type="file"]') as HTMLInputElement;

        if(input) {
            input.click();
        }
    }
```

Where `.drop-element` is the dropzoner per

```svelte
<Dropzone containerClasses={`drop-element${dragged || files.accepted.length > 0 || files.rejected.length > 0 ? '' : ' hidden'}`} on:drop={dropHandler} disableDefaultStyles={true} on:dragenter={dragEnterHandler} on:dragleave={dragLeaveHandler}>
```

But yeah, it works quite well!

**Comment by arackaf** [comment]
Gah - sorry! I promise I'll get to this tomorrow.

**Comment by arackaf** [comment]
This should be released in 2.0.8

**Comment by roberrrt-s** [comment]
That's awesome. Thank's a lot!
